src/main/java/net/sf/mzmine/util/spectraldb/entry/DBEntryField.java

```diff
@@ -201,7 +201,7 @@ public String getNistMspID() {
    */
   public static DBEntryField forMspID(String key) {
     for (DBEntryField f : values()) {
-      if (f.getNistMspID().equals(key))
+      if (f.getNistMspID().equalsIgnoreCase(key))

```

Comment:
Yes, that was important as it was not really consistent in the msp libraries I have tested

Reply:
I recommend to add a comment there that explains why IgnoreCase is important. Otherwise some rogue developer might accidentally change it back in the future.

Reply:
done